src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/String/OrdinalStringFrozenDictionary_LeftJustifiedSingleCharCaseInsensitive.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Following naming convention of other files already committed, shouldn't this be `OrdinalStringFrozenDictionary_LeftJustifiedCaseInsensitiveSingleChar.cs`?